blueflood-elasticsearch/src/main/resources/metrics_mapping.json

```diff
@@ -13,8 +13,9 @@
                 "index": "not_analyzed"
             },
             "metric_name": {
-                "type": "string",
-                "index": "not_analyzed"
+                "index_analyzer": "prefix-test-analyzer",
```

Comment:
This is used by elastic search server which gets spun up during the ElasticIO tests, so whenever we send metrics to it, it will use this analyzer to analyze the metric fields and thus store the tokens along with the entire field string.